handler/src/main/java/io/netty/handler/ssl/JdkAlpnApplicationProtocolNegotiator.java

```diff
@@ -20,21 +20,11 @@
 /**
  * The {@link JdkApplicationProtocolNegotiator} to use if you need ALPN and are using {@link SslProvider#JDK}.
  */
+// TODO(nmittler): Can we make this package-private?
```

Comment:
IIRC initially we didn't have the extra `ApplicationProtocolConfig` layer to enable creation of this internally. Strictly speaking this would be a breaking change, but I'm not against it ... @normanmaurer ?